common/frame_session.go

```diff
@@ -322,6 +317,8 @@ func (fs *FrameSession) parseAndEmitWebVitalMetric(object string) error {
 		tags = tags.With("url", wv.URL)
 	}
 
+	tags = tags.With("rate", wv.Rating)
```

Comment:
Should it be `rate` or `rating`?

Reply:
Yeah, `rating` is better, not sure why i chose `rate` considering we're already using `rating` elsewhere. Done in 76f3b0fc45b343ad666fb1ff480ccfb09397d7fe

Reply:
To be honest I was looking for some reference on Web Vitals doc for it, but found none, but still, as you say, we have been using `rating` so it seems more correct :+1: